src/org/labkey/test/WebDriverWrapper.java

```diff
@@ -2286,19 +2292,23 @@ public boolean isElementVisible(Locator loc)
         return loc.findElement(getDriver()).isDisplayed();
     }
 
-    public void assertElementPresent(Locator loc)
+    public WebElement assertElementPresent(Locator loc)
     {
-        assertTrue("Element is not present: " + loc.getLoggableDescription(), isElementPresent(loc));
+        Optional<WebElement> optionalElement = loc.findOptionalElement(getDriver());
+        assertTrue("Element is not present: " + loc.getLoggableDescription(), optionalElement.isPresent());
+        return optionalElement.orElse(null);
```

Comment:
I think this is a good change. I'm wondering if this begins to deviate from the function name. I'm also wondering about the use of the optionalElement, if the assert passes shouldn't we just return the web element, no need for the .orElse(null).

Reply:
I thought IntelliJ would compain if I just called `optionalElement.get()` but its actually smart enough to know that `assertTrue(optionalElement.isPresent())` makes that a safe call.
Fixed.